org.eclipse.winery.lsp/src/main/java/org/eclipse/winery/lsp/Server/ServerCore/ToscaTextDocService.java

```diff
@@ -0,0 +1,63 @@
+package org.eclipse.winery.lsp.Server.ServerCore;
+
+import org.eclipse.lsp4j.*;
+import org.eclipse.lsp4j.services.TextDocumentService;
+import org.eclipse.winery.lsp.Server.ServerAPI.API.context.BaseOperationContext;
+import org.eclipse.winery.lsp.Server.ServerAPI.API.context.ContextBuilder;
+import org.eclipse.winery.lsp.Server.ServerAPI.API.context.LSContext;
+import org.eclipse.winery.lsp.Server.ServerCore.Utils.CommonUtils;
+
+import java.nio.file.Path;
+
+public class ToscaTextDocService implements TextDocumentService {
+    private final LSContext serverContext;
+
+    public ToscaTextDocService(LSContext serverContext) {
+        this.serverContext = serverContext;
+    }
+    @Override
+    public void didOpen(DidOpenTextDocumentParams params) {
+        MessageParams messageParams = new MessageParams();
+        messageParams.setMessage("A file was opened!");
+        messageParams.setType(MessageType.Info);
+        this.serverContext.getClient().logMessage(messageParams);
+        
+        Path uriPath = CommonUtils.uriToPath(params.getTextDocument().getUri());
+        BaseOperationContext context = ContextBuilder.baseContext(this.serverContext);
+        if (uriPath.toFile().getName().endsWith(".txt")) {
```

Comment:
I think, following could also work (and is shorter):

```suggestion
        if (uriPath.toString().endsWith(".txt")) {
```

Reply:
Please stick with `.yaml`. No need for `.txt` workarounds.

We should be able to manage to create example / test TOSCA files using the `.yaml` extension.

However, one needs to support `.yml` and `.tosca`, too. - YAML has two file extensions and typically, tosca files are named `.tosca`.